finagle-base-http/src/test/scala/com/twitter/finagle/http/DefaultHeaderMapTest.scala

```diff
@@ -16,12 +16,12 @@ class DefaultHeaderMapTest extends AbstractHeaderMapTest with ScalaCheckDrivenPr
   final def newHeaderMap(headers: (String, String)*): HeaderMap = DefaultHeaderMap(headers: _*)
 
   test("apply()") {
```

Comment:
can we push all of these tests down into AbstractHeaderMapTest?

Reply:
I really shouldn't read review comments bottom to top